java/client/src/main/java/glide/api/commands/StringCommands.java

```diff
@@ -0,0 +1,14 @@
+package glide.api.commands;
+
+import glide.api.models.commands.SetOptions;
+import java.util.concurrent.CompletableFuture;
+
+/** String Commands interface to handle single commands that return Strings. */
```

Comment:
`set` without options returns nothing.
```suggestion
/** String Commands interface to handle commands perform string manuplation. */
```